content/source/docs/enterprise/install/active-active.html.md

```diff
@@ -456,18 +456,17 @@ You can now safely change the number of instances in your Auto Scaling Group ( o
 
 #### Disable the Admin Dashboard
 
-Before you scale up, disable the Admin dashboard as mentioned earlier in this guide. Add the `disable-replicated-ui` flag when you call the install script, as such:
+Before scaling beyond the first node, you must disable the  as mentioned earlier in this guide. This is done by adding the `disable-replicated-ui` flag as a parameter when you call the install script, as such:
```

Comment:
Looks like the "admin dashboard" phrase just got dropped here. Also, the title of this section should be adjusted for consistency.

```suggestion
#### Disable the Replicated Admin Console

Before scaling beyond the first node, you must disable the Replicated Admin Console as mentioned earlier in this guide. This is done by adding the `disable-replicated-ui` flag as a parameter when you call the install script, as such:
```

Reply:
I think the one edit got hosed in my last commit on this topic. The section title was missed. Both corrected. (couldn't use yoour suggested commit as due to the hosing, it thinks there are deleted lines and doesn't support that??)